src/Main.hx

```diff
@@ -147,7 +147,7 @@ class Main {
 				return workspaceFolder.uri.fsPath;
 			}
 		}
-		return null;
+		return Vscode.workspace.workspaceFolders[0].uri.fsPath; // try first workspace folder
```

Comment:
checkstyle should only happen for files within your workspace, so `null` is ok or even a requirement for files that were opened from outside (e.g. when you follow a type or method into your haxelib folder)